monitor/internal/pod/monitor.go

```diff
@@ -183,8 +183,8 @@ func (m *PodMonitor) Run(ctx context.Context) error {
 	case err := <-errCh:
 		return fmt.Errorf("pod: %s", err.Error())
 	case <-time.After(5 * time.Second):
-		// we give the controller 5 seconds to report back
-		return errors.New("pod: controller did not start within 5s")
+		// we give the controller 5 seconds to report back before we issue a warning
```

Comment:
Can we also log how many seconds did it take to start the controller.

Reply:
maybe this help us in future debugging.

Reply:
good idea ... I'll add that as a debug log.

Reply:
looks like this now:

```
DEBUG	pod: controller startup finished	{"duration": "100.176139ms"}
```